blazar/plugins/base.py

```diff
@@ -79,6 +80,22 @@ def on_start(self, resource_id):
         """Wake up resource."""
         pass
 
+    def list_resource_properties(self, property_name, query, detail):
```

Comment:
Good idea implementing this on the base plugin, definitely saves some boilerplate.

Reply:
removed.